### DISCUSSION TITLE
Manjaro

### DISCUSSION THREAD

**Discussion by DIMFLIX** [prasanthrangan/hyprdots#138] [Q&A]
Will this build work on Linux Manjaro? Are there any known issues? If yes, can you tell me the solution?

**Comment by T-Crypt** [comment]
Depends on the Manjaro iso you are using.. 

I believe the Manjaro minimal install (which is recommended for this) is only available for ARM. So you would probably have to install Manjaro and have a bunch of GUI packages and conflicts involved in getting this to work as Manjaro provides a DE from install.

Why would you use Manjaro when you can just install minimalist Arch and install the dotfiles? Just curious

**Reply by DIMFLIX** [reply]
The problem is that I'm not a particularly experienced Linux user. Of course I know how to google, and I know some things. But it seems like arch is a difficult distribution for a beginner.

**Reply by T-Crypt** [reply]
You can install Debian and use Senshi111's version mentioned in the README.md 

Debian is a little more user friendly / commonly used for application packages so you wouldn't have to rely on AUR to install what you need.

**Reply by Senshi111** [reply]
I only tested in Debian sid not the stable or testing version, so im not sure is it works on stable and testing. But its easy to upgrade debian to sid, just go to /etc/apt/source.list and change repo name from bookworm to sid.
After that just run sudo apt update && sudo apt full-upgrade

**Reply by Gedankenn** [reply]
I'm not a beginner, but I've always used Debian as my Linux distribution. The best decision of my life was to switch to Arch. The community documentation is spot on, and the AUR is amazing. Things that I would struggle to install on Debian are just a command away on Arch.

Also i'm a heavly gammer, and i still have to find a game that cant run on arch, i'm actually playing fallout newvegas with mods from nexus, not even a single crash i had.

**Reply by Senshi111** [reply]
Me for example don't like AUR at all. I also maintained some packages that i needed for work, it's kind a boring. Other thing why i moved away from arch is unstableness from time to time. I don't have time to fixing stuff from time to time.
Only thing that i miss sometimes from arch is pacman and that's all

**Reply by T-Crypt** [reply]
@Senshi111 speaking of unstableness waybar-hyprland-git got packaged with spdlog-git and broke the dependency on spdlog and bricked my package manager as I couldn't update anything so I had to remove waybar-hyprland-git and update my packages, and then install waybar package.

had like 90 AUR package updates and took so freaking long to update

**Reply by prasanthrangan** [reply]
> waybar-hyprland-git got packaged with spdlog-git and broke the dependency on spdlog

oh noooo... not again :(

**Reply by Senshi111** [reply]
@T-Crypt yes i know. It happens to me like 10 days ago or something, because of that i said, "ok that's enough" so i switch back to debian sid and tried to make those theme to work on debian sid
